src/OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -283,12 +281,10 @@ public void SetInputValue(string key, object value)
 				numberInput.Value = Convert.ToSingle(val);
 				return;
 			}
-			var separator = GetSeparator(key);
-			if (separator != null)
+			if (GetSeparator(key) == null)
```

Comment:
Should Separator even be considered as input if null is returned regardless? Maybe just completely remove that check